design.css

```diff
@@ -111,14 +111,12 @@ tr:nth-child(even) {
 	text-align: center;
 }
 
-#fadeEffect{
-    animation: fadeIn 3s;
-}
-@keyframes fadeIn {
-    from {
-        opacity:0;
-    }
-    to {
-        opacity:1;
-    }
-}
\ No newline at end of file
+.footer{
+	grid-column-start: 1;
+	grid-column-end: 5;
+	background-color: gray;
+	height: 150px;
+	text-align: center;
+}
+
+.topLink{cursor: pointer;}

```

Comment:
One thing bad here is why did you include the style change in this PR? In this example is trivial but when you add unrelated changes in a PR. sometimes reverting commits in future will cause unintended results.